res/values-es/strings.xml

```diff
@@ -76,74 +76,74 @@
     <string name="PrefGroupSearch">Búsqueda</string>
     <string name="PrefGroupStories">Historias</string>
     <string name="PrefGroupTor">Tor</string>
-    <string name="PrefMainFontSize">Tamaño de Letra</string>
+    <string name="PrefMainFontSize">Tamaño de letra</string>
     <string name="PrefNever">Nunca</string>
-    <string name="PrefRate">Calificar Aplicación</string>
+    <string name="PrefRate">Calificar aplicación</string>
     <string name="PrefReadable">Legibilidad</string>
     <string name="PrefReadableSummary">Encendido, cuando disponible</string>
     <string name="PrefRecentSearch">Recientes</string>
     <string name="PrefRecordCookies">Guardar cookies</string>
-    <string name="PrefRecordHistory">Guardar recientes</string>
+    <string name="PrefRecordHistory">Guardar Recientes</string>
     <string name="PrefRegion">Región</string>
-    <string name="PrefResetSources">Restaurar Fuentes</string>
+    <string name="PrefResetSources">Restaurar fuentes</string>
     <string name="PrefSavedItems">Favoritos</string>
-    <string name="PrefSearchResults">Resultados de Búsqueda</string>
-    <string name="PrefSendFeedback">Deja Retroalimentación</string>
+    <string name="PrefSearchResults">Resultados de búsqueda</string>
+    <string name="PrefSendFeedback">Dejar retroalimentación</string>
```

Comment:
```Dejar retroalimentación``` sounds weird although it makes sense. Fundeu recommends 'Enviar sugerencias'
http://www.fundeu.es/recomendacion/feedback-en-espanol-respuesta-reaccionesimpresiones-retorno-932/